examples/flutter_dynamic_form/lib/bloc/new_car_bloc.dart

```diff
@@ -71,7 +74,8 @@ class NewCarBloc extends Bloc<NewCarEvent, NewCarState> {
     );
   }
 
-  Stream<NewCarState> _mapYearChangedToState(YearChanged event) async* {
+  Stream<NewCarState> _mapNewCarYearChangedToState(
+      NewCarYearChanged event) async* {
```

Comment:
```suggestion
      NewCarYearChanged event,) async* {
```

Reply:
should I refactor to add trailing commas throughout examples or are you suggesting just this one?

Reply:
I wouldn’t worry about it actually